### PR TITLE
Correct makefile known parameters and how we handle them

### PR DESCRIPTION
**Pull request by cjeanner**
As a pull request owner and reviewers, I checked that:
- [X] Appropriate testing is done and actually running
- [X] Appropriate documentation exists and/or is up-to-date:
  - [X] README in the role
  - [X] Content of the docs/source is reflecting the changes

### CODE REVIEW

**Comment by cjeanner** [comment]
recheck

**Comment by cjeanner** [comment]
recheck

**Comment by cjeanner** [comment]
recheck

**Comment by cjeanner** [comment]
recheck

**Comment by cjeanner** [comment]
Over the past rechecks, all of the jobs passed at least once... Currently, the baremetal is failed, but all of the others are green. Will run a recheck to ensure it's really a one-off.

**Comment by cjeanner** [comment]
recheck

**Comment by cjeanner** [comment]
/hold

Unsure about the cause of the failure in baremetal. In any cases, I'd rather triple-check with @pablintino once he's around.

File: ci_framework/roles/install_yamls/defaults/main.yml [hunk 1]
```diff
@@ -30,8 +31,8 @@ cifmw_install_yamls_repo: "{{ cifmw_installyamls_repos | default(ansible_user_di
 cifmw_install_yamls_tasks_out: "{{ cifmw_install_yamls_out_dir }}/roles/install_yamls_makes/tasks"
 cifmw_install_yamls_whitelisted_vars:
   - DEPLOY_DIR
-  - METALLB_POOL
   - KUBECONFIG
+  - METALLB_POOL
```
Comment (pablintino) [inline ci_framework/roles/install_yamls/defaults/main.yml:35]: Any single new variable added here is a smell, and this one was no exception, used temporary but no one removed it :(

**Review by pablintino — COMMENTED** [review]
We can remove `METALLB_POOL` is this commit, the rest is +2 from my side.

File: ci_framework/roles/install_yamls/defaults/main.yml (hunk 1 above)
Comment (cjeanner) [reply ci_framework/roles/install_yamls/defaults/main.yml]: :+1:

**Comment by pablintino** [comment]
recheck

**Comment by pablintino** [comment]
recheck

**Comment by pablintino** [comment]
Seems that Zuul failures are unrelated. Force merging it to speed up the CI stabilization efforts.

**Pull request merged**